client/my-sites/domains/domain-management/domain-connect/domain-connect-authorize.jsx

```diff
@@ -31,8 +31,17 @@ class DomainConnectAuthorize extends Component {
 		const { providerId, serviceId, params, translate } = this.props;
 		const { domain } = params;
 
-		wpcom
-			.getDnsTemplateRecords( domain, providerId, serviceId, params )
+		wpcom.req
+			.post(
+				'/domains/' +
```

Comment:
This is a good opportunity to use template literals. I think it would be more readable on a single line.

Reply:
👍